actionpack/lib/abstract_controller/caching/fragments.rb

```diff
@@ -135,6 +135,7 @@ def expire_fragment(key, options = nil)
       end
 
       def instrument_fragment_cache(name, key) # :nodoc:
+        return unless ActionView::Base.enable_fragment_cache_logging
```

Comment:
I found that fragment caching's log is not at debug level, so we can't early return at the place you said. And I think we should do that in here instead.